packages/main/cypress/specs/Dialog.cy.tsx

```diff
@@ -82,6 +84,34 @@ describe("Keyboard", () => {
 	});
 });
 
+describe("Initial Focus", () => {
+	it("Tokenizer focusing", () => {
+		cy.mount(
+			<>
+				<Dialog id="dialogId" headerText="Tokens">
+					<Tokenizer id="tokenizer">
+						<Token text="Token 1" id="token1"/>
+						<Token text="Token 2" id="token2"/>
+					</Tokenizer>
+				</Dialog>
+			</>
+		);
+
+		// Wait tokenizer item navigation to be initialized
+		// eslint-disable-next-line cypress/no-unnecessary-waiting
+		cy.wait(100);
```

Comment:
Replace with 
```tsx
cy.get("#token1")
			.shadow()
			.find(".ui5-token--wrapper")
			.should("have.attr", "tabindex", "0");
```

or completely replace Tokenizer and Tokens with html elements, one of which with style display: contents